tests/test_database.py

```
@@ -0,0 +1,96 @@
+
+import os
+from unittest.mock import MagicMock, patch
+
+import pytest
+from app.database import Base, engine, init_db
+from app.database_migrations import run_startup_migrations
+from sqlalchemy import inspect
+
+
+@pytest.fixture(autouse=True)
+def clean_database_after_each_test(db_session):
```

Ruff (0.14.14)

[warning] Unused function argument: `db_session`

(ARG001)